README.md

```diff
@@ -132,7 +132,7 @@ build all the tests.   There are quite a few tests so this does take a significa
 that is not necessary if you want to experiment with changes.   You can submit the building
 of the tests with the 'skiptests' argument to the build script.
 
-Thus to get a build as quickly as possible type the following (using \ as the directory separator, use / on Unix machines)
+Thus to get a build as quickly as possible type the following (using `\` as the directory separator, use `/` on Unix machines)
 ```bat
     .\build skiptests 
```

Comment:
Seems like the leading four spaces were used previously when `` ```bat \n* ``` ``  wrapper wasn't there; they can be removed.